common/source/docs/common-camera-runcam.rst

```diff
@@ -0,0 +1,64 @@
+
+.. image:: ../../../images/camera-runcam-fc-setup.jpg
+    :target:  ../_images/camera-runcam-fc-setup.jpg
+    :width: 450px
+
+  On some camera-only devices the pinout is labelled such that TX should be connected to TX and RX to RX. If your RunCam device is not recognized try reversing the signal wires. Not all cameras support a UART ground pin - they expect to get ground from the power supply.
+  The camera must be in UART control mode. On split-style cameras this should be the default. On camera-only devices you will need to follow the procedure for putting the device in UART control mode. Consult your RunCam manual for more details.
+  Note that so-called "Joystick control" is not supported and some devices only support joystick control, so be careful when selecting the RunCam model you are going to use. The following devices have been tested successfully - RunCam Split 3s Micro, RunCam Split 2s, RunCam Racer 3.
+
+* In order to recognize the RunCam serial protocol on the UART, set the UART serial protocol to 26. So for instance for serial port 5, set :ref:`SERIAL5_PROTOCOL <SERIAL5_PROTOCOL>` 26 (= RunCam protocol). Adjust the serial port's protocol parameter to 26, for the serial port that your RunCam's serial wires are connected to, as required. The correlation between serial port numbering and UART physical ports for your autopilot should be documented in its description page linked :ref:`here <common-autopilots>`.
+  When the camera is connected correctly and the serial port configured correctly you will see a message similar to the following at boot: ``APM: RunCam initialized, features 0x0077, 2-key OSD`` otherwise you will see ``APM: RunCam device not found``.
+  The camera must be powered and fully booted in order to accept commands. This can take some time and the delay between the flight controller booting and the camera being probed can be controlled through :ref:`CAM_RC_BT_DELAY <CAM_RC_BT_DELAY>`. The camera will be probed repeatedly until :ref:`CAM_RC_BT_DELAY <CAM_RC_BT_DELAY>` elapses, but once that has expired the camera will not be probed again.
+
+* Some split-style cameras support three modes - video standby, video start and picture mode. It is not possible to tell from the RunCam device protocol whether this is the type of camera in use and so this must be configured. Set :ref:`CAM_RC_DEVICE_TYPE <CAM_RC_DEVICE_TYPE>` 2. All other cameras should be fine with the default.
+
+* In order to control start and stop of video recording from a radio transmitter set :ref:`RC8_OPTION <RC8_OPTION>` = 78 to enable triggering from transmitter switch 8. Adjust the switch number for triggering on different switches. If the switch is in the low position then video recording is stopped, if in the high position then video recording is started.
```

Comment:
not transmitter switch 8, RC input channel 8
not switch number....channel number in second sentence

Reply:
Fixed